sql/moz-fx-data-shared-prod/bigeye_derived/collection_service_v1/metadata.yaml

```diff
@@ -10,6 +10,7 @@ labels:
   table_type: aggregate
 scheduling:
   dag_name: bqetl_bigeye_derived
+  arguments: ['--schema_update_option=ALLOW_FIELD_ADDITION']
```

Comment:
These are argument for the query.py.  The script doesn't accept this argument so this shouldn't be here. You would want to put this in the in the `LoadJobConfig` but it's already doing this it's `WRITE_TRUNCATE` on an unpartitioned table.